utils/scan_utils.py

```diff
@@ -421,6 +421,11 @@ def build_scan_options_parser() -> ArgumentParser:
     parser.add_argument("--workers", nargs=1,
                         help="Limit parallel threads per-scanner to a number.")
     # TODO: Should workers have a default value?
+    parser.add_argument("--no-fast-cache", action="store_true", help="".join([
+        "Do not use fast caching even if a scanner supports it.  This option ",
+        "will cause domain-scan to use less memory, but some scans may be ",
```

Comment:
Nit - it's not that some `scans` may be repeated, in the sense that a scan is a domain-scan construct of "taking an action on a hostname", but it may cause some network activity (or other expensive actions) to be needlessly repeated.

Reply:
Change committed.